data/graph_db_utils.py

```diff
@@ -0,0 +1,44 @@
+import os
+from dotenv import load_dotenv
+from neo4j import GraphDatabase
+
+load_dotenv()
+
+URI = os.getenv("NEO4J_URI")
+AUTH = (os.getenv("NEO4J_USERNAME"), os.getenv("NEO4J_PASSWORD"))
+
+driver = GraphDatabase.driver(URI, auth=AUTH)
+
+def test_connection():
+    try:
+        driver.verify_connectivity()
+        print("database connection active")
```

Comment:
Can we replace all `print` statements with logging methods. We can use `debug` level if they are lower priority messages that we may want to filter out when reviewing log messages.